model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -6,10 +6,17 @@
 # Please, refer to the LICENSE file in the root directory.
 # SPDX-License-Identifier: BSD-3-Clause
 import functools
-from typing import Any, Callable, Literal, Mapping, Optional, Sequence, TypeAlias, TypeVar
+from typing import (
+    Any,
+    Callable,
+    Literal,
+    Mapping,
+    Optional,
+    Sequence,
+    TypeAlias,
+    TypeVar,
+)
```

Comment:
question: since this is becoming a long list, should we switch to import typing directly and then use e.g. `typing.Any` ?

Reply:
I don't see the problem with the long list,the imports are folded away by most IDE anyway.

Reply:
ok then we leave it :)